postgres/datadog_checks/postgres/metadata.py

```diff
@@ -301,20 +301,21 @@ def report_postgres_extensions(self):
         elapsed_s = time.time() - self._time_since_last_extension_query
         if elapsed_s >= self.pg_extensions_collection_interval and self._collect_extensions_enabled:
             self._extensions_cached = self._collect_postgres_extensions()
-        event = {
-            "host": self._check.reported_hostname,
-            "database_instance": self._check.database_identifier,
-            "agent_version": datadog_agent.get_version(),
-            "dbms": "postgres",
-            "kind": "pg_extension",
-            "collection_interval": self.collection_interval,
-            "dbms_version": payload_pg_version(self._check.version),
-            "tags": self._tags_no_db,
-            "timestamp": time.time() * 1000,
-            "cloud_metadata": self._check.cloud_metadata,
-            "metadata": self._extensions_cached,
-        }
-        self._check.database_monitoring_metadata(json.dumps(event, default=default_json_event_encoding))
+        if self._extensions_cached:
```

Comment:
hm I see this going two ways
1. We're resending cached events. Possibly duplicate unintentionally
2. We need to be sending these duplicated events so that they don't expire out on the backend from the resource ttl. 

Is the goal for option 2 here? Wondering if that's why we're wrapping `_<resource>_cached` vs just nesting this under the collection config enabled

Reply:
The answer should probably go in the "motivation" section for future readers.

Reply:
Yes, we have to resend to avoid TTLing in REDAPL

Reply:
Updated to
1. Send on the collection interval
2. Send the correct collection interval to base the TTL off of